src/features/pages/partners/deriv-prime/browse-our-faq/data.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react'
+import CommercialPlan from '../commercial-plan'
+import { DerivPrimeFAQDataItem } from './types'
+
+export const generalFAQData: DerivPrimeFAQDataItem[] = [
```

Comment:
can you change this variable name to `snake_case` something like `general_faq_data`

Reply:
Sure, but is there a specific reason for this change?

Reply:
yes all data varaiable should be in `snake_case`